lib/web_ui/lib/src/engine/pointer_binding.dart

```diff
@@ -728,11 +733,9 @@ mixin _WheelEventListenerMixin on _BaseAdapter {
     if (_debugLogPointerEvents) {
       print(event.type);
     }
+    // [ui.PointerData] has a binding to the native `event` so users can choose
+    // to prevent default (or not).
     _callback(e, _convertWheelEventToPointerData(event));
-    // Prevent default so mouse wheel event doesn't get converted to
-    // a scroll event that semantic nodes would process.
-    //
-    event.preventDefault();
```

Comment:
Removing this will break our wheel handling until the Framework changes land :/

Reply:
Yeah, the best solution is to continue doing preventDefault by default unless allowed by the user. I'll flip the API around.